batchflow/models/tf/mobilenet.py

```diff
@@ -219,8 +222,203 @@ def block(cls, inputs, filters, residual=False, strides=1, expansion_factor=6, w
+                       strides=1, **kwargs)
+        x = depthwise_conv(x, kernel_size=kernel_size, strides=strides,
+                           padding='same', data_format=kwargs['data_format'], name='%s-depthwise' % name)
+        x = conv_block(x, 'na', name='%s-na' % name, **kwargs)
+
+        if se_block:
+            x = cls.se_block(x, num_filters // 4, name='%s-se' % name, data_format=kwargs['data_format'],
+                             activation=[kwargs.get('activation', tf.nn.relu), h_sigmoid])
+
+        x = conv_block(x, 'cn', filters, 1, name='%s-down' % name, **kwargs)
+
+        if residual:
+            x = inputs + x
+        return x
+
+
+_V3_SMALL_DEFAULT_BODY = [
+    dict(repeats=1, filters=16, expansion_factor=1, strides=2, kernel_size=3, se_block=True),
+    dict(repeats=1, filters=24, expansion_factor=4.5, strides=2, kernel_size=3, se_block=False),
+    dict(repeats=1, filters=24, expansion_factor=3.7, strides=1, kernel_size=3, se_block=False,
+         residual=True),
+    dict(repeats=1, filters=40, expansion_factor=4, strides=2, kernel_size=5, activation=h_swish, se_block=True),
+    dict(repeats=2, filters=40, expansion_factor=6, strides=1, kernel_size=5, activation=h_swish, se_block=True,
+         residual=True),
+    dict(repeats=2, filters=48, expansion_factor=3, strides=1, kernel_size=5, activation=h_swish, se_block=True),
+    dict(repeats=3, filters=96, expansion_factor=6, strides=2, kernel_size=5, activation=h_swish, se_block=True),
+]
+
+class MobileNet_v3_small(MobileNet_v3):
+    """ MobileNet version 3 small architecture """
+    @classmethod
+    def default_config(cls):
+        config = MobileNet_v3.default_config()
+        config['initial_block'].update(dict(layout='cna', filters=16, kernel_size=3, strides=2, activation=h_swish))
+        config['body'].update(dict(width_factor=1, layout=deepcopy(_V3_SMALL_DEFAULT_BODY)))
+        config['head'].update(dict(layout='cnavcacV', filters=[576, 1280, 2], pool_size=7,
+                                   kernel_size=1, activation=h_swish, se_block=True,
+                                   se_kwargs=dict(activation=[h_swish, h_sigmoid], ratio=144)))
+        return config
+
+    @classmethod
+    def head(cls, inputs, se_kwargs=None, name='head', **kwargs):
+        """ The last network layers which produce predictions
+
+        Parameters
+        ----------
+        inputs : tf.Tensor
+            input tensor
+        name : str
+            scope name
+
+        Returns
+        -------
+        tf.Tensor
+        """
+        kwargs = cls.fill_params('head', **kwargs)
+        layout = kwargs.pop('layout')
+        filters = kwargs.pop('filters')
+        if kwargs.get('se_block'):
+            x = conv_block(inputs, layout=layout[:3], filters=filters[0], name='%s-conv1' % name, **kwargs)
+            x = cls.se_block(x, **se_kwargs, data_format=kwargs['data_format'], name='%s-se' % name)
```

Comment:
You don't need many args for the same need. `se_block` is enough for everything.